### DISCUSSION TITLE
Please correct video at these timestamps

### DISCUSSION THREAD

**Discussion by ayesparshh** [Cyfrin/foundry-full-course-cu#2078] [Q&A]
![Screenshot 2024-07-22 203608](https://github.com/user-attachments/assets/ea4c0c79-8cc0-4477-80dc-55d7d41088af)
![Screenshot 2024-07-22 203540](https://github.com/user-attachments/assets/24df0cfc-eb77-411a-b98e-e9418c01595f)
![Screenshot 2024-07-22 203556](https://github.com/user-attachments/assets/4e462744-f082-4390-aea6-ce60ab127aba)

**Comment by EngrPips** [comment]
Thanks for pointing this out, @ayesparshh, but people usually report stuff like this in the #issue tab.
